2021/24xxx/CVE-2021-24964.json

```diff
@@ -1,18 +1,76 @@
 {
```

Comment:
An exception of type JSONDecodeError occurred. Arguments:

Reply:
Please check change requests and resubmit.

Reply:
This has been done already: https://github.com/CVEProject/cvelist/pull/3912/commits/70aabc675881c14c020059a4c0c63346db6bafdd